ui/lib/components/ActionsButton/index.tsx

```diff
@@ -21,11 +21,6 @@ export default function ActionsButton({
     throw new Error('actions should at least have one action')
   }
 
-  // hide actions if the task does not succceed
-  if (disabled) {
```

Comment:
@breeswish recommended to hide the actions if the task does not succeed. It would be better to make the behavior consistent with the one on manual profiling. But you can still keep the display way until PMs want to change it.

Reply:
OK, got it.